templates/hpc/base_hpc.html

```diff
@@ -0,0 +1,7 @@
+{% extends "templates/base.html" %}
```

Comment:
Can you rebase against main branch? I created this same file but named `_base_hpc.html` (notice the underscore, to make it consistent with the other templates). Thanks